docs/src/components/GuidelinesBuilder/GuidelinesPageBuilder.js

```diff
@@ -65,28 +83,39 @@ const Section = (props: SectionProps) => {
           level="h2"
           className={getClassName('bpk-docs-guidelines-page__heading')}
         >
-          {title}
+          <BpkText textStyle="xxl" bold>
```

Comment:
Is this going to be applicable to all the new pages? We might be able to just pass the textStyle and bold into the Heading as i believe it uses BpkText under the hood?

Reply:
Yes all pages but that is the design so I don't see as a problem. I does't use BpkText, just style it using the mixins so passing `xxl` and `bold` won't work. I think this is the less `intrusive` way of doing this without affecting other pages